client/jetpack-connect/main.jsx

```diff
@@ -314,12 +315,7 @@ class JetpackConnectMain extends Component {
 	}
 
 	isInstall() {
-		/*
-		 * FIXME: `return ! this.props.type` should be sufficient
-		 * I'm avoiding significantly changing this implementation
-		 * until propTypes have been around for a while.
-		 */
-		return includes( [ 'install', 'pro', 'premium', 'personal' ], this.props.type );
+		return includes( FLOW_TYPES, this.props.type );
```

Comment:
I think this existing logic is wrong here, which is why the url entry button says _Install_ for paid flows instead of just the install flow. I'll tackle it in a different PR.

Reply:
Seems like it

Reply:
Seems it was intentional in #6460